.github/workflows/repo2cpe-download.yaml

```diff
@@ -0,0 +1,37 @@
+name: Download Repo2CPE
+on:
+  push:
+    branches:
+    - "yli3/**"
+  schedule:
```

Comment:
we can update this repo2cpe mapping every 7 days, what you think? @RTann  @jvdm

Reply:
yeah let's do daily